roles/ceph-rgw-loadbalancer/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# You can override vars by using host or group vars
+
+###########
+# GENERAL #
+###########
+
+haproxy_frontend_port: 80
```

Comment:
```suggestion
haproxy_frontend_port: 80
#
#virtual_ips:
#   - 192.168.238.250
#   - 192.168.238.251
#
#virtual_ip_netmask: 24
#virtual_ip_interface: ens33
```

then you just have to run `./generate_group_vars_sample.sh` and the diff should disappear in the corresponding sample file.

Reply:
this is a good way, thanks, I'll update soon.